### ISSUE TITLE
Does the code in the OCC generation section constitute match content in paper?

### ISSUE THREAD

**Issue by Orbis36** [Arlo0o/UniScene-Unified-Occupancy-centric-Driving-Scene-Generation#550]
I'm reproducing the semantic occupancy generation results with the released weights (Occupancy_Generation_ckpt_DiT_0600000.pt, run_eval_dit.sh → eval_OccDiT.py), and I ran into two conditioning inputs in the eval/training code that appear to be derived from the ground-truth occupancy. Since at genuine "BEV layout → occupancy" inference time the GT occupancy is not available, I'd like to confirm whether I'm reading this correctly, because it seems to differ from how the paper describes the method.

What the paper says. As I understand it, the paper states the Occupancy DiT generates the occupancy latent sequence from noise volumes conditioned on the BEV layouts, i.e. the conditioning is the BEV layout and the diffusion starts from standard Gaussian noise. I could not find any mention of a noise prior, reference-frame noise initialization, a temporal anchor, or scene-density / "meta" conditioning.

What the code does.

1. Noise prior initialized from GT occupancy — eval_OccDiT.py, main():
z_ori = OccVAE.encode(occ_ori) * scale_factor          # occ_ori is the GT occupancy
z_ori_s = z_ori[:, :, ref_idx].unsqueeze(2).repeat(1, 1, Tframe, 1, 1)
z = torch.randn((bz, in_ch, Tframe, 50, 50), device=device) + lambda_np * z_ori_s
1. with --lambda_noise_prior=0.05 in run_eval_dit.sh (and 0.03 in run_train_dit.sh). Since scale_factor=70 normalizes the latent to ~unit std, the initial noise carries roughly a few percent of the GT reference-frame latent, and the same anchor is repeated across all frames.
2. meta conditioning computed from GT occupancy — eval_OccDiT.py, main():
new_meta = occ_meta[:, ref_idx]                        # cal_occ_meta(GT occ): density scores
model_kwargs = dict(y=y, meta=new_meta, cfg_scale=args.cfg_scale)
2. where cal_occ_meta (in dataload_util.py) computes global density statistics (total occupied ratio, terrain / manmade / vegetation / other densities) from the GT occupancy. The training loop (train_OccDiT.py) uses the same occ_meta[:, ref_idx] and the same GT-based noise prior (noise_prior = torch.randn_like(x) + lambda_np * z_ref, z_ref = encoded GT reference frame).

Questions.

1. Are the reported occupancy-generation metrics (mIoU, F3D/FID, MMD) produced with these two GT-derived inputs (the lambda_noise_prior term and the GT-based occ_meta) enabled, exactly as in run_eval_dit.sh?
2. If so, how should one interpret the "generate occupancy from BEV layout" claim, given that both the noise prior and occ_meta are functions of the ground-truth occupancy that would be unavailable at inference?
3. What is the intended way to run a pure BEV-layout → occupancy inference with the released weights (no GT information)? Specifically:
  - Is setting lambda_noise_prior=0 (pure Gaussian noise) the intended layout-only mode?
  - Since the released weights were trained with meta conditioning, what should meta be at true inference time when GT is unavailable — a constant / dataset-average vector, zeros, or is a meta-free checkpoint available?
4. Is occ_meta meant to be an auxiliary conditioning signal that is part of the method (and if so, could you point to where it's described in the paper), or is it a training/eval convenience?

I want to make sure I compare fairly against your numbers and don't accidentally disable something intended. Thanks in advance for the clarification!

**Comment by XunhengKuang** [comment]
I have encountered the same issue as well. It seems that the released code does not fully match the method described in the paper.

Also, have you noticed that the rendered semantic maps and depth maps are of very poor quality? Please see #551 for more details.

I am also trying to reproduce this work, and I would greatly appreciate your response so that we can discuss these issues and exchange our findings.
